common/lib/dependabot/shared_helpers.rb

```diff
@@ -234,13 +205,39 @@ def self.configure_git_credentials(credentials)
           "@#{cred.fetch('host')}"
 
         git_store_content += authenticated_url + "\n"
+        configure_git_to_use_https(cred.fetch('host'))
       end
 
       # Save the file
       File.write("git.store", git_store_content)
     end
     # rubocop:enable Metrics/PerceivedComplexity
 
+    def self.configure_git_to_use_https(host)
```

Comment:
Feel free to:

```suggestion
    def self.configure_git_to_use_https(host) # rubocop:disable Metrics/AbcSize
```

I don't think that abstracting any of this code is useful or even an improvement